Library/Formula/gumbo-parser.rb

```diff
@@ -0,0 +1,24 @@
+class GumboParser < Formula
+  homepage "https://github.com/google/gumbo-parser/blob/master/README.md"
+  url "https://github.com/google/gumbo-parser/archive/v0.9.3.tar.gz"
+  sha256 "038354c0394591b270eafc9203aef4516afbf78a66fcb2c4fff5180593b003f0"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool"  => :build
+
+  def install
+    system "./autogen.sh"
+
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "pkg-config", "--cflags", "--libs", "gumbo"
```

Comment:
Can a test be added to do something with the tool itself rather than just leaning on pkgconfig to see if it exists? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!